webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/inventory/PostCommits.scala

```diff
@@ -175,3 +177,10 @@ class TriggerPolicyGenerationPostCommit[A](
      } else ZIO.unit) *> records.succeed
   }
 }
+
+class TriggerInventoryScorePostCommit[A](scoreServiceManager: ScoreServiceManager) extends PostCommit[A] {
```

Comment:
there is no need to add that one if there is already an `CoreNodeFact` event handler. 
Here, it's really only for the low level things related to inventory translation

Reply:
Hum, the fact that the `InventoryScoreEvent` is triggered on inventory update (here or in nodefactrepository) means that we have to thing about the migration story to have score correctly init after a rudder upgrade. 
I doesn't seems trivial either, if we don't want for it to be too costly at boot, but something like "look if table score exist but is empty" is not enough, since it can fail in the middle. Right now I don't see how to do that but with a flag "migration finish" that can be quick checked at boot, and have a long migration where we look for each node without a system update score and compute it.

Reply:
I think we can wait for an inventory

Reply:
discussed online, it will be better to have the score init as soon as possible to make user WOUAH, but clearly not blocking for that PR.